amp_entrypoint.sh

```diff
@@ -3,4 +3,5 @@ sudo mkdir -p /opt/splunk/lib/python2.7/site-packages
 sudo mkdir -p /opt/splunk/lib/python3.7/site-packages
 sudo pip install -r requirements.txt --target=/opt/splunk/lib/python2.7/site-packages
 sudo pip install -r requirements.txt --target=/opt/splunk/lib/python3.7/site-packages
+sudo pip install -r requirements.txt
```

Comment:
instead of installing the requirements in 3 separate locations, can you install it in just 1 location and then create a symlink?
```suggestion
sudo pip install -r requirements.txt --target=/opt/splunk/lib/python2.7/site-packages --install-option="--install-scripts=$PWD/bin" -t python_modules/
ln -s /opt/splunk/lib/python2.7/site-packages /opt/splunk/lib/python3.7/site-packages
```
the ` --install-option="--install-scripts=$PWD/bin" -t python_modules/` is for [specifying the location](https://stackoverflow.com/a/39155112) to install the binaries

Reply:
I like this idea, the only problem I think that might occur is that afaik, Splunk will still install it's regular packages in both the 2.7 and 3.7 site-packages folders, and there might be problems if it tries to install them all in the same place. I'll play around with it a bit though and see if I can get it to work.

I didn't know about that `install-scripts` flag though, that's awesome